### ISSUE TITLE
Is the code wrong in the networks.py?

### ISSUE THREAD

**Issue by Med-Process** [senguptaumd/Background-Matting#55]
Is there some errors or specific setting in the line 99 (networks.py)?
`oth_feat=torch.cat([self.comb_back(torch.cat([img_feat,back_feat],dim=1)),self.comb_seg(torch.cat([img_feat,seg_feat],dim=1)),self.comb_multi(torch.cat([img_feat,back_feat],dim=1))],dim=1)`

May be it should be self.comb_multi(torch.cat([img_feat,multi_feat],dim=1))],dim=1))?

**Comment by senguptaumd** [comment]
Check issue #36.
